specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -304,43 +322,84 @@ _jakarta.xml.rpc.handler.MessageContext_ type.]
-Jakarta EE expert group follows a process first defined by the Java SE
-expert group ( _http://mreinhold.org/blog/removing-features_ ) to prune
+Jakarta EE Platform Specification Project follows a process similar to the one first defined by the Java SE
+expert group ( _http://mreinhold.org/blog/removing-features_ ) to stabilize and remove
 technologies from the platform in a careful and orderly way that
 minimizes the impact to developers using these technologies, while
-allowing the platform to grow even stronger. In short, the process
-defines two steps:
+allowing the platform to grow even stronger. In short, our process
+defines three steps:
+
+. The Platform Specification Project for release N 
+of the platform decides to propose that a particular feature be
+marked "proposed optional". The specification for that release documents the proposal.
+. The Platform Specification Project for release N+1 decides whether to
+mark the feature as "optional" for this N+1 release, retain it as a required component,
+or leave it in the "proposed optional" state for the next N+2 release cycle to decide.
+. Eventually, the Platform Specification Project for release N+2 (or beyond) can decide
+to officially "remove" the "optional" feature from the Platform.
 
+The result of successfully applying this
+policy to a feature is to allow a gradual removal
+of the feature as a required component of the platform.
+Product vendors can _choose_ to remove or include an "optional" feature in their
+products.
+Although a "removed" feature can still be included in a vendor's product, vendors 
+need to realize that the "removed" feature is no longer defined as part of the Platform
+and, thus, the integration of this "removed" feature with the rest of the Platform
+is left as an exercise for the vendor.
+
+Technologies that are "proposed optional" are marked Proposed Optional (POPT) in
+<<a2159, Jakarta EE Technologies>>.
+Technologies that are "optional" as of Jakarta EE 9 are marked Optional (OPT) in
+<<a2159, Jakarta EE Technologies>>.
+Technologies that are "removed" from the Jakarta EE Platform are documented in 
+<<a2333, Removed Jakarta Technologies>>.
+
+*Note:* In order to get on a level playing field, Jakarta EE 9 took a couple of liberties
+with the application of "proposed optional", "optional", and "removed" technologies per
+the Jakarta EE 9 Release Plan, available at _https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan_.  
+Going forward, it is the expectation that Jakarta EE
+releases will follow these defined policies.
+
+[[a2333]]
+==== Removed Jakarta Technologies
+Jakarta EE 9 introduced the concept of "removed" technologies.
+This is the final stage of a technology's lifecycle where the technology is
+officially *removed* from the Jakarta EE Platform.
+This is a stronger statement than making a technology "optional", since a "pruned"
+technology will no longer be maintained for future versions of the Platform.
+
```

Comment:
'"pruned" technology' must be '"removed" technology'

Reply:
Nice catch, @kazumura!  Thank you!